feat: Update operator docs (#858)
vegaprotocol/documentation: docs/node-operators/get-started/setup-datanode.md

The default value of the `huge_pages` config is `try`. Setting it to `off` usually reduces the RAM usage, however, it increases the CPU usage on the machine.

#### Max locks per transaction

New Value:

```conf
max_locks_per_transaction = 256
```

> This value may change in future to bigger number. It depends on the traffic on the network. See [the official docs](https://postgresqlco.nf/doc/en/param/max_locks_per_transaction/) for reference.

Reviewer: ```suggestion
This value may change in the future to a bigger number, depending on the network traffic. See the [Postgres docs ↗](https://postgresqlco.nf/doc/en/param/max_locks_per_transaction/) for reference.
```

> :::note

Reviewer: ```suggestion
:::note Error state
```

> When the value is too low, you may end with the following error: `ERROR: out of shared memory (SQLSTATE 53200)`.

Reviewer: ```suggestion
When the value is too low, you may see the following error: `ERROR: out of shared memory (SQLSTATE 53200)`.
```
